build.sh

```diff
@@ -108,10 +144,15 @@ curl -fsSL -o OPEN_SOURCE_LICENSES.TXT "${OPEN_SOURCE_LICENSES_URL}"
 for config in "Debug" "Release"; do
   INSTALL_DIR="${INSTALL_DIR_PREFIX}${config}"
   mkdir -p "${INSTALL_DIR}/bin"
+  cp "temp/build-${config}/src/shadertrap/shadertrap" "${INSTALL_DIR}/bin/"
 
   case "$(uname)" in
   "Linux")
-    cp "temp/build-${config}/src/shadertrap/shadertrap" "${INSTALL_DIR}/bin/"
+    ;;
+
+  "MINGW"*|"MSYS_NT"*)
+    # Required as EGL is not available on Windows in a standard way.
+    cp "${HOME}/angle-release/libEGL.dll" "${INSTALL_DIR}/bin/"
```

Comment:
Will this work? My guess is you would need the rest of ANGLE as well. I would omit initially, and test locally.

Reply:
This is indeed how I have gotten it to work locally.